checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -41,22 +57,23 @@
     extends GenericAnnotatedTypeFactory<CFValue, LockStore, LockTransfer, LockAnalysis> {
 
     /** Annotation constants */
-    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE;
+    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE, GUARDEDBYINACCESSIBLE, GUARDEDBY, GUARDEDBYBOTTOM, JCIPGUARDEDBY, JAVAXGUARDEDBY, GUARDSATISFIED;
```

Comment:
I'll split it into 3 lines.
The aliasing mechanism doesn't work perfectly under all circumstances. Most importantly, I need the 3 @GuardedBys (javax, jcip and our own) to be subtypes of one another provided that their values are identical. I use them in LockQualifierHierarchy.isGuardedBy below. isGuardedBy in turn is used by isSubtype.